website_cookie_notice/readme/ROADMAP.rst

```diff
@@ -0,0 +1,16 @@
+* If you are using this module in version < 8.0.2.0.0 and update it, any other
```

Comment:
If this information relevant to this version?

Reply:
No, I just removed it